qemu/Makefile

```diff
@@ -0,0 +1,123 @@
+	m4 -D VERSION=$* $< > user-data.yaml
+	cloud-localds $@ user-data.yaml
+
+ubuntu-noble-amd64-ocaml-%.qcow2: noble-server-cloudimg-amd64.qcow2 seed-%.iso
+	qemu-img create -f qcow2 -b $< -F qcow2 $@ 20G
+	qemu-system-x86_64 -m 16G -smp 8 -machine accel=kvm,type=pc -cpu host -display none -vnc :0 \
+		-drive file=$@ \
+		-drive file=seed-$*.iso,format=raw \
+		-nic user,hostfwd=tcp::60022-:22
+
+ubuntu-noble-riscv64-ocaml-%.qcow2: noble-server-cloudimg-riscv64.qcow2 seed-%.iso
+	qemu-img create -f qcow2 -b $< -F qcow2 $@ 20G
+	qemu-system-riscv64 -m 16G -smp 8 -machine type=virt -nographic \
+		-bios /usr/lib/riscv64-linux-gnu/opensbi/generic/fw_jump.bin \
+		-kernel /usr/lib/u-boot/qemu-riscv64_smode/uboot.elf \
+		-drive file=$@,if=virtio \
+		-drive file=seed-$*.iso,format=raw,if=virtio \
+		-netdev user,id=net0,hostfwd=tcp::60022-:22 -device virtio-net-device,netdev=net0
+
+.NOTINTERMEDIATE: $(wildcard noble-server-cloudimg-*.qcow2)
+
+noble-server-cloudimg-%.qcow2: noble-server-cloudimg-%.img
+	curl -C - -L https://cloud-images.ubuntu.com/noble/current/$< -o $@
+
+noble-server-cloudimg-%.img: ;
+
+# busybox
+
+seed.iso: busybox.yaml
+	cp $< user-data.yaml
+	cloud-localds $@ user-data.yaml
+
+busybox.qcow2: noble-server-cloudimg-amd64.qcow2 seed.iso
+	qemu-img create -f qcow2 -b $< -F qcow2 $@ 20G
+	qemu-system-x86_64 -m 16G -smp 8 -machine accel=kvm,type=pc -cpu host -display none -vnc :0 \
+		-drive file=$@ \
+		-drive file=seed.iso,format=raw \
+		-nic user,hostfwd=tcp::60022-:22
+
+# OpenBSD
+
+openbsd: openbsd-76-amd64-ocaml-4.14.2.qcow2 openbsd-76-amd64-ocaml-5.2.0.qcow2
+
+openbsd-76-amd64-ocaml-%.qcow2: tftp/auto_install tftp/bsd.rd tftp/etc/boot.conf mirror%
+	qemu-img create -f qcow2 $@ 24G
+	qemu-img create -f qcow2 cache-ffs.qcow2 20G
+	python3 -m http.server --directory mirror --bind 0.0.0.0 80 & \
+	qemu-system-x86_64 -m 16G -smp 8 -machine accel=kvm,type=pc -cpu host -display none -vnc :0 \
+		-drive file=$@,if=virtio \
+		-drive file=cache-ffs.qcow2,if=virtio \
+		-nic user,model=virtio,tftp-server-name=10.0.2.2,tftp=tftp,bootfile=auto_install,hostfwd=tcp::60022-:22 ;\
+	kill \%1
+
+tftp/auto_install:
+	mkdir -p tftp
+	curl -L https://cdn.openbsd.org/pub/OpenBSD/7.6/amd64/pxeboot -o tftp/auto_install
+
+tftp/bsd.rd:
+	mkdir -p tftp
+	curl -L https://cdn.openbsd.org/pub/OpenBSD/7.6/amd64/bsd.rd -o tftp/bsd.rd
+
+tftp/etc/boot.conf:
+	mkdir -p tftp/etc
+	echo boot tftp:/bsd.rd > tftp/etc/boot.conf
+
+mirror%: install.site.m4 install.conf disklabel
+	m4 -D VERSION=$* $< > install.site
+	chmod +x install.site
+	mkdir -p mirror/pub/OpenBSD/7.6/amd64
+	tar -czf mirror/pub/OpenBSD/7.6/amd64/site76.tgz install.site
+	for f in BUILDINFO SHA256.sig base76.tgz bsd bsd.mp bsd.rd comp76.tgz game76.tgz man76.tgz pxeboot xbase76.tgz xfont76.tgz xserv76.tgz xshare76.tgz ; do curl -C - -L https://cdn.openbsd.org/pub/OpenBSD/7.6/amd64/$$f -o mirror/pub/OpenBSD/7.6/amd64/$$f ; done
+	cd mirror/pub/OpenBSD/7.6/amd64 && ls -l > index.txt
```

Comment:
It is included in the OpenBSD distribution.  For example, see https://cdn.openbsd.org/pub/OpenBSD/7.6/amd64/index.txt

The [FAQ](https://www.openbsd.org/faq/faq4.html) says it should be updated when a custom `site76.tgz` is added.  Therefore, I think `ls -l` is the expected format.

>Note: If you intend to provide the sets over HTTP(s), place site76.tgz in your source directory and include it in your `index.txt`. It will then be an option at install time.